### DISCUSSION TITLE
Utility classes with BEM

### DISCUSSION THREAD

**Discussion by EddyVinck** [getbem/getbem.github.io#243] [Q&A]
I just started reading about BEM and I quite like the idea. I do have a question though:

Where does BEM stand on utility classes?

At work we like to use classes like `.margin-bottom` and `.margin-bottom-large` but I can't find anything about this on the website.  The reason we use those kind of classes is because a lot of the margins (in this  case) will be the same for a lot of elements. 

Thanks.

**Comment by Ygohel18** [comment]
.margin__bottom
.margin__bottom--large

**Comment by tadatuta** [comment]
@EddyVinck you should use [mixes](https://en.bem.info/methodology/css/#mixes) with special blocks for that:

```html
<div class="my-block margin-bottom"></div>
```

or even better use variables in styles instead of extra markup.
